test/app.e2e-spec.ts

```diff
@@ -0,0 +1,31 @@
+import { Test } from '@nestjs/testing';
```

Comment:
I think the filename should follow the same convention as the other e2e tests.

Reply:
@ahmedolaibrahim  It should not. This is a test file while other files are test suite files. We have only one file with test that runs multiple test suite files. The reason for that  is to have only one instance of app running during tests because we are listening on port 3000. We could run each test one by one but it would be very time consuming.

Reply:
You want to name it `app.e2e.spec.ts`?

Reply:
yeah i think `auth.e2e.spec.ts` is more suitable.